common/modules/models/Task.ts

```diff
@@ -42,6 +42,9 @@ export interface IGeoCode {
 export interface IComment {
   id: string
   text: string
+  datetime: Date
```

Comment:
use the camelCase rule for variable initialization.

Reply:
I just used the same style as in the rest of the application

for example:
export interface ITaskModel {
  _id?: ObjectId
  ...
  dateofcreate: Date                       // dateOfCreate
  deadline: string                            // deadLine
  customer?: string
  taskexecutors: [ItaskExecutors]    // taskExecutor
  ...
}

Reply:
This is bad practice. We should stick to the accepted style, especially since it improves the readability of the variables. Therefore, it will be necessary to refactor the front-end part.